torch/_dynamo/test_minifier_common.py

```diff
@@ -133,7 +133,7 @@ def _maybe_subprocess_run(self, args, *, isolate, cwd=None):
                 stderr.getvalue().encode("utf-8"),
             )
         else:
-            return subprocess.run(args, capture_output=True, cwd=cwd)
+            return subprocess.run(args, capture_output=True, cwd=cwd, check=True)
```

Comment:
Would be good to add a comment why we think output is not checked there (and perhaps refactor the function to return string, rather than an object...

Reply:
I can set it back to false, I just wanted to address all the errors. It's only in test code so not that vital so I can set it to false or noqa it since we aren't sure.

Reply:
Okay, the test suite revealed it should be check=False